tests/rootless.sh

```diff
@@ -100,11 +100,10 @@ function enable_cgroup() {
 		chmod g+rwx "$CGROUP_MOUNT/$CGROUP_PATH"
 		chmod g+rw "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.subtree_control" "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.procs" "$CGROUP_MOUNT/cgroup.procs"
 		# Fix up cgroup.type.
-		echo threaded > "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.type"
 		# Make sure cgroup.type doesn't contain "invalid". Otherwise write ops will fail with ENOTSUP.
 		# See http://man7.org/linux/man-pages/man7/cgroups.7.html
 		if grep -qw invalid "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.type"; then
-			exit 1
+			echo threaded > "$CGROUP_MOUNT/$CGROUP_PATH/cgroup.type"
```

Comment:
So, if we fixed this in `CreateCgroupPath`, there is no more need to set it here, or am I missing something?